app/models/bot/slack.rb

```diff
@@ -25,23 +25,21 @@ def should_notify?(target, model)
   end
 
   def notify_slack(model)
-    p = self.get_project(model)
-    t = self.get_team(model, p)
+    t = self.get_team(model)
 
     if self.should_notify?(t, model)
       webhook = t.setting(:slack_webhook)
-      channel = p.setting(:slack_channel) unless p.nil?
```

Comment:
I realize we are losing ther per-project Slack notification customization - that's fine for now because we will revisit this logic later for the MTAN project